src/lib/pubkey/pk_algs.cpp

```diff
@@ -419,6 +429,12 @@ std::unique_ptr<Private_Key> load_private_key(const AlgorithmIdentifier& alg_id,
    }
 #endif
 
+#if defined(BOTAN_HAS_CLASSICMCELIECE) || defined(BOTAN_HAS_CLASSICMCELIECE)
+   if(alg_name.starts_with("mceliece")) {
```

Comment:
Uses ClassicMcEliece elsewhere we should be consistent

Reply:
We named the algorithm identifiers as defined in the specifications:
<img src="https://github.com/user-attachments/assets/a125bb89-1efd-4d97-834a-73fcc1668f51" width="150">

The OIDs were taken from the [IETF Hackathon GitHub](https://github.com/IETF-Hackathon/pqc-certificates/blob/master/docs/oid_mapping.md#nist-round-4-kem-algorithm-oids). IMO, we should stay with the same naming. But I agree that it is annoying that these are not named `classic...`